README.md

```diff
@@ -1,13 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Mobile Portfolio!
+Mobile Version Portfolio!
 
 > Description the project.
-> In this project I created My Mobile Portfolio Version.
+> In this project I created My Mobile Portfolio Version.I have used CSS Grid and Flexbox.
 
 ## Built With
 
-Html and css
+Html and CSS
+CSS Grid and Flexbox
 
 To get a local copy up and running follow these simple example steps.
```

Comment:
- [ OPTIONAL]  As per the project description, I think it would be better to use this [template](https://github.com/microverseinc/readme-template) provided by microverse. :thinking: 

_This requirement is optional for now since the previous reviewer didn't catch it, but I strongly recommend to work on it_